connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/MsSQLServerDatabaseOnDocker.scala

```diff
@@ -28,5 +28,6 @@ class MsSQLServerDatabaseOnDocker extends DatabaseOnDocker {
   override val jdbcPort: Int = 1433
 
   override def getJdbcUrl(ip: String, port: Int): String =
-    s"jdbc:sqlserver://$ip:$port;user=sa;password=Sapass123;"
+    s"jdbc:sqlserver://$ip:$port;user=sa;password=Sapass123;" +
+      "encrypt=true;trustServerCertificate=true"
```

Comment:
This is enforced by default from now?

Reply:
Yeah, I referred and confirmed with ms-jdbc docs, which says it's recommended for testing purposes. I didn't investigate thoroughly when this change for default values was applied due to the large version span.